src/frontends/pytorch/src/op/var.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_var(NodeContext& context) {
+    auto data = context.get_input(0);
+    bool unbiased = true;
+    bool keepdims = false;
+    auto num_elements = numel(context, 0);
+    bool keepdim_mean;
+    std::shared_ptr<ov::Node> mean;
+    ov::Output<ov::Node> axes;
+    if (context.inputs().size() == 2) {
```

Comment:
Please add comment about what is the difference between inputs size == 2 and inputs size != 2.

Reply:
added comments regarding var operation signature